configuration/metrics.nix

```diff
@@ -70,6 +75,22 @@
         domain = "${routes.metrics.subDomain}.${routes.domain}";
         root_url = "https://${routes.metrics.subDomain}.${routes.domain}";
       };
+
+      auth = {
+        signout_redirect_url = "https://${routes.authentik.subDomain}.${routes.domain}/application/o/${routes.metrics.applicationSlug}/end-session/";
+        oauth_auto_login = true;
+      };
+      "auth.generic_oauth" = {
+        name = "authentik";
+        enabled = true;
+        client_id = "MuFTPc2aPXUUoL6UKJQJ1cx35Ie5isHVG8cssUqV";
+        client_secret = " $__file{${config.sops.secrets.grafanaClientSecret.path}}";
+        scopes = "openid email profile";
+        auth_url = "https://${routes.authentik.subDomain}.${routes.domain}/application/o/authorize/";
+        token_url = "https://${routes.authentik.subDomain}.${routes.domain}/application/o/token/";
+        api_url = "https://${routes.authentik.subDomain}.${routes.domain}/application/o/userinfo/";
+        role_attribute_path = "contains(groups, 'Grafana Admins') && 'Admin' || contains(groups, 'Grafana Editors') && 'Editor' || 'Viewer'";
```

Comment:
The path we are currently using is:
```
"contains(groups, 'grafana Admins') && 'Admin' || 'Viewer'"
```
If we need Editors we can add them in the future. Also notice the lowercase g on grafana.